relation_engine/batchload/delta_load.py

```diff
@@ -47,68 +56,133 @@ def load_graph_delta(
          specified.
     """
     db = database
+    if merge_source and not db.get_merge_collection():
+        raise ValueError('A merge source is specified but the database ' +
+           'has no merge collection')
+    _process_verts(db, vertex_source, timestamp, load_version)
+    if merge_source:
+        _process_merges(db, merge_source, timestamp, load_version)
     
-    # count = 0
-    for v in vertex_source:
-        # could batch things up here if slow
-        dbv = db.get_vertices([v[_ID]], timestamp).get(v[_ID])
-        if not dbv:
-            db.save_vertex(v[_ID], load_version, timestamp, v)
-        elif not _special_equal(v, dbv):
-            db.expire_vertex(dbv[_KEY], timestamp - 1)
-            db.save_vertex(v[_ID], load_version, timestamp, v)
-        else:
-            # mark node as seen in this version
-            db.set_last_version_on_vertex(dbv[_KEY], load_version)
-        # count += 1
-        # if count % 1000 == 0:
-        #     print(f'node {count}')
+    if _VERBOSE: print(f'expiring vertices: {time.time()}')
```

Comment:
Maybe use the logging lib here in the future

Reply:
Yeah, `_VERBOSE` stuff is temporary AFAIC. Long term I'd add a callback so the caller can respond to events like these.